### PR TITLE
pkg: file stat warning

### PR DESCRIPTION
**Pull request by spzala**
Provide warning and doc instead of enforcing file permission.

### CODE REVIEW

**Comment by spzala** [comment]
@jingyih @gyuho @xiang90 - please take a look to the changes. Thanks!

File: Documentation/op-guide/security.md [hunk 1]
```diff
@@ -431,6 +431,9 @@ No. etcd doesn't encrypt key/value data stored on disk drives. If a user need to
 * Let client applications encrypt and decrypt the data
 * Use a feature of underlying storage systems for encrypting stored data like [dm-crypt]
 
+### I’m seeing a log warning that "directory X exist without desired file permission -rwx------"
+When etcd create certain new directories it sets file permission to 700 to prevent unprivileged access as possible. However, if user has already created a directory with own preference, different than 700, etcd uses the existing directory and logs the warning message.
```
Comment (spzala) [inline Documentation/op-guide/security.md:435]: @jingyih we can doc it differently or modify the warning below per review comments. Thanks!

**Comment by hexfusion** [comment]
This makes a lot of sense thanks for making the change @spzala

**Comment by xiang90** [comment]
lgtm

**Pull request merged**